src/frontend/src/components/forms/fields/RelatedModelField.tsx

```diff
@@ -32,6 +36,8 @@ export function RelatedModelField({
     fieldState: { error }
   } = controller;
 
+  const form = useFormContext();
```

Comment:
Does this have any drawback on performance? Eg does this component now gets rerendered more often as without that hook?